.github/workflows/build-and-deploy.yaml

```
@@ -0,0 +1,56 @@
+name: build and deploy mtb
```

Since this workflow is used by multiple app workflows (ARC, Open Bridge), should we use a generalized name? 

```suggestion
name: build and deploy app
```

---

Good point, changed.